res_company_search_view/tests/test_res_company.py

```diff
@@ -0,0 +1,104 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestResCompany(TransactionCase):
```

Comment:
Sorry just saw this on https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0

> Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.

```suggestion
from odoo.addons.base.tests.common import BaseCommon


class TestResCompany(BaseCommon):
```

Reply:
@Christian-RB yes! it's Fixed.please review.